antenna-documentation/src/site/markdown/template-bundle-development.md

```diff
@@ -4,6 +4,24 @@ automatically when no other bundle is configured. This guide will show how to cr
 integrate it in the Antenna workflow run. 
 
 ## Custom Template Bundle
+
+### How to create the template PDF files
+
+First, the template has to be developed with a tool like Microsoft Word / Open Office / Libre Office. The corresponding 
+document will consist of 4 pages which will represent the 4 templates PDF files:
+* cover.pdf
+* copyright.pdf 
+* content.pdf 
+* back.pdf
+
+The page size is A4 at 72 PPI with 595x842 pixels. The cover.pdf file will be the cover page of the attribution document 
+which will contain the project name and version starting at 80x547. The copyright.pdf file will be the second page of 
```

Comment:
Ah, so the contents of the cover page are filled in at a fixed *pixel* position? I always thought the pages would need to contain special makers, like `$REPLACE_ME_WITH_THE_PROJECT_NAME_AND_VERSION`, which get replaced with the contents...

Reply:
How do you control the font properties (type, size, style) of the text being inserted?

Reply:
> The font type will be loaded from the pdf file

Ok, so that hardens my assumption that the cause of the [font issues](https://github.com/oss-review-toolkit/ort/issues/2755) we're seeing are caused by not all characters of the font being embedded into the PDF.

> but the size and style are hardcoded.

Please add this bit to the docs then.